pages/index.js

```diff
@@ -57,6 +58,13 @@ export default class Home extends React.Component{
     }, RefreshTimeInSeconds * 1000);
   }
 
+  checkStorageForCategory = () => {
+    const selectedCategory = window.localStorage.getItem('ddf-category');
+    if(selectedCategory){
```

Comment:
Please go for short circuit evaluation such as `selectedCategory && this.setState({ selectedCategory })`

Reply:
Done